### PR TITLE
Update links to new Grid/Graph view instead of relying on redirects

### PR DESCRIPTION
**Pull request by ashb**
Recently we changed the URL strucutre for Graph and Grid (nee Tree)
views, but there were still a few places that used the old names (which
still work thanks to the redirects we have in place.)

This changes the front end to use the new URLs directly.

A few of the places the URL is generated/tweaked via JS, so we needed a
pass a "fake" dag_id -- `$dag_id` to the `url_for()` helper (otherwise
it would blow up) and then replace it directly in the string from JS.

### CODE REVIEW

File: airflow/www/templates/airflow/dags.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
   <meta name="last_dag_runs_url" content="{{ url_for('Airflow.last_dagruns') }}">
   <meta name="dag_stats_url" content="{{ url_for('Airflow.dag_stats') }}">
   <meta name="task_stats_url" content="{{ url_for('Airflow.task_stats') }}">
-  <meta name="tree_url" content="{{ url_for('Airflow.tree') }}">
+  <meta name="grid_url" content="{{ url_for('Airflow.grid', dag_id='${dag_id}') }}">
```
Comment (bbovenzi) [inline airflow/www/templates/airflow/dags.html]: Why do we refer to `${dag_id}` here and `$dag_id` in `dags.js`? Is that a typo?

Comment (ashb) [reply airflow/www/templates/airflow/dags.html]: Whoops yes typo!

Comment (ashb) [reply airflow/www/templates/airflow/dags.html]: Fixed to `__DAG_ID__~ as `$` and `{` are both URL escaped so those characters made it more complex to handle

**Review by bbovenzi — APPROVED** [review]

**Comment by bbovenzi** [comment]
LGTM. But some www tests are failing

**Comment by ashb** [comment]
Fixed :)

**Comment by ashb** [comment]
Random MSSQL failure (db wasn't ready)

**Pull request merged**